src/components/videoConference/WaitingRoom.tsx

```diff
@@ -51,7 +51,7 @@ export const WaitingRoom = ({
 	};
 
 	const confirmButton: ButtonItem = {
-		label: translate('anonymous.waitingroom.dataProtection.button'),
+		label: translate('videoConference.waitingroom.dataProtection.button'),
```

Comment:
I think the waitingRoom is a feature for "Anonymous", it's a bit sad, that Debbi did not remove that as I think that's something we can delete.